HotTips/Tips/EDI_UnreachableCode.md

```diff
@@ -0,0 +1,3 @@
+﻿Remove code that will never be executed by pressing **Ctrl+.** with your caret anywhere in the faded out code that is unreachable. 
```

Comment:
...and then what? That only opens a lightbulb.

" and choose 'Remove unused code' from the Quick Actions list" etc.